nemo_curator/utils/fuzzy_dedup_utils/shuffle_utils.py

```diff
@@ -151,7 +153,9 @@ def get_shuffle_partition_info(
     return shuffle_part_ids
 
 
-def text_bytes_aware_shuffle(df, partition_on, text_column, num_workers=None):
+def text_bytes_aware_shuffle(
+    df, partition_on: str, text_column: str, num_workers: Optional[int] = None
+) -> Optional[None]:
```

Comment:
Thanks for adding annotations in many parts of the codebase that's missing them 😄